src/custom-element.js

```diff
@@ -813,6 +813,19 @@ function createBaseCustomElementClass(win) {
       if (newWidth !== undefined) {
         setStyle(this, 'width', newWidth, 'px');
       }
+
+      let newLayout = this.layout_;
+      if (newHeight !== undefined) {
+        newLayout = Layout.FIXED_HEIGHT;
+        if (newWidth !== undefined) {
```

Comment:
This may not be correct, as multiple calls can be made to change `width` or `height`. The checks here should be for `height` or `width` inline styles.